### DISCUSSION TITLE
Help a newbie out

### DISCUSSION THREAD

**Discussion by ctrtrends** [ml-explore/mlx#194] [General]
Sorry I have a stupid question.

I'm really interested in the field so I tried following the documentation (https://ml-explore.github.io/mlx/build/html/index.html).

I figured out the building and installing mlx. But in the quick start section and further... where does it expect me to put and run the code?

**Comment by awni** [comment]
That's all python code running in a terminal. If you open your terminal and type `python` then you will get a python REPL which you can input code into.

I suggest reading up a bit more on how to use python as we assume that as a prereq for using MLX.
